specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -0,0 +1,641 @@
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "status": {
+          "description": "Get or set the status of model.",
+          "enum": [ "created", "ready", "invalid" ],
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Get or set the created date time of the model.",
+          "type": "string"
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Get or set the model last updated datetime.",
+          "type": "string"
+        }
+      }
+    },
+    "ModelsResult": {
+      "description": "Result of query operation to fetch multiple models.",
+      "type": "object",
+      "properties": {
+        "models": {
+          "description": "Collection of models.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ModelResult"
+          }
+        }
+      }
+    },
+    "FormDocumentReport": {
+      "type": "object",
+      "properties": {
+        "documentName": {
+          "description": "Reference to the data that the report is for.",
+          "type": "string"
+        },
+        "pages": {
+          "format": "int32",
+          "description": "Total number of pages trained on.",
+          "type": "integer"
+        },
+        "errors": {
+          "description": "List of errors per page.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "status": {
+          "description": "Status of the training operation.",
+          "enum": [
+            "success",
+            "partialSuccess",
+            "failure"
+          ],
+          "type": "string"
+        }
+      }
+    },
+    "FormOperationError": {
+      "description": "Error reported during an operation.",
+      "type": "object",
+      "properties": {
+        "errorMessage": {
+          "description": "Message reported during the train operation.",
+          "type": "string"
+        }
+      }
+    },
+    "ErrorResponse": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorInformation"
+        }
+      }
+    },
+    "ErrorInformation": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "innerError": {
+          "$ref": "#/definitions/InnerError"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
+    },
+    "InnerError": {
+      "type": "object",
+      "properties": {
+        "requestId": {
+          "type": "string"
+        }
+      }
+    },
+    "AnalyzeResult": {
+      "description": "Analyze API call result.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Status of the analyze operation.",
+          "enum": [
```

Comment:
@ramparab have you considered using x-ms-enum instead and then set modelAsString: true

Reply:
We would love to but at this point its bit of bigger change than we want to take on as this swagger contains two 'Status' enums with different values which is causing issues.